workitem/field_definition_blackbox_test.go

```diff
@@ -6,32 +6,71 @@ import (
 	"testing"
 
 	"github.com/fabric8-services/fabric8-wit/resource"
-	. "github.com/fabric8-services/fabric8-wit/workitem"
+	"github.com/fabric8-services/fabric8-wit/workitem"
 )
 
-func TestListFieldDefMarshalling(t *testing.T) {
-	t.Parallel()
-	resource.Require(t, resource.UnitTest)
-	def := FieldDefinition{
-		Required:    true,
-		Label:       "Salt",
-		Description: "Put it in your soup",
-		Type: ListType{
-			SimpleType:    SimpleType{Kind: KindList},
-			ComponentType: SimpleType{Kind: KindString},
-		},
-	}
+func testFieldDefinitionMarshalUnmarshal(t *testing.T, def workitem.FieldDefinition) {
 	bytes, err := json.Marshal(def)
 	if err != nil {
 		t.Errorf(err.Error())
 		return
 	}
 
-	t.Logf("bytes are ", string(bytes))
-	unmarshalled := FieldDefinition{}
+	t.Log("bytes are ", string(bytes))
```

Comment:
you could probably use `t.Logf"bytes are %s", bytes)` here (but that's ok)

Reply:
Good catch. Fixed in c40b0b2e